src/planscape/plan/tests.py

```diff
@@ -804,6 +804,138 @@ def test_good_multipolygon(self):
         self.assertEqual(ProjectArea.objects.count(), 1)
 
 
+class BulkCreateProjectAreaTest(TransactionTestCase):
```

Comment:
nit: are there any tests for checking behavior when user=None, project.owner = None?

Reply:
Done